.jules/sentinel.md

```
@@ -0,0 +1,12 @@
+## 2026-02-24 - SQL Injection and Sensitive Data Leakage in Logs
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)